core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -129,6 +131,9 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
 
   private val pendingReplayTasksCount = new java.util.concurrent.atomic.AtomicInteger(0)
 
+  /** filesystem metrics: visible for test access */
```

Comment:
Sorry I can't find that. The only reference to `.metrics` I see in this PR is for the app cache metrics, not `FsHistoryProvider`. If you can provide an actual link to the line you're referring to, that would help.